js/script.js

```diff
@@ -92,10 +92,18 @@ jqueryNoConflict(document).ready(function() {
 
   fetchActionKitData(args.actionkit, function(actionKitLayerGroups) {
     $.extend(layers, actionKitLayerGroups);
-    Object.keys(layers).forEach(function(campaignName) {
+    Object.keys(actionKitLayerGroups).forEach(function(campaignName) {
       icons[campaignName] = getEventIcon();
     });
-    initializeTabletopObject(root_spreadsheet);
+
+    fetchControlShiftData(args.controlshift, function(controlShiftLayerGroups) {
+      $.extend(layers, controlShiftLayerGroups);
+      Object.keys(controlShiftLayerGroups).forEach(function(category) {
+        icons[category] = getPetitionIcon();
+      });
+
+      initializeTabletopObject(root_spreadsheet);

```

Comment:
Hmm.  I'm starting to get nervous about load times here.  If I'm following correctly, the sequence is now:

1. Fetch all actionkit layers in parallel; when all have been fetched,
2. Fetch all control shift layers' p1 in parallel; when all have been fetched, 
3. Fetch all control shift layers' remaining pages in parallel; when all have been fetched,
4. Fetch the google spreadsheet data through tabletop; when that has been fetched,
5. Load and display the map layers

Is that correct?

If so, this seems like it could start to significantly delay things, especially now that we're including a data source with pagination.  I think the ideal sequence would be more like:

1. In parallel:
* fetch the google spreadsheet data through tabletop
* fetch all actionkit layers
* fetch all control shift layers' p1 (and then all subsequent pages in parallel when the p1s are done)
2. As soon as we've finished fetching the google spreadsheet data, we will know exactly what layers we need to include on the map (since all the other data sources specify their layers in configuration before any external http requests are made) so we can draw the legend.
3. As each data source responds, draw its markers on the map, and recalculate clustering

That said, this would obviously be a substantial refactoring, so I think it makes sense to merge this PR without it.